ios/engine/KMEI/KeymanEngine/Classes/Manager.swift

```diff
@@ -835,16 +827,6 @@ public class Manager: NSObject, UIGestureRecognizerDelegate {
       return .none
     }
   }
-  
-  // Keyboard download notification observers
-  private func keyboardDownloadCompleted(_ keyboards: [InstallableKeyboard]) {
-    // There's little harm in reloading the keyboard (and thus, KMW) for a clean reset
-    // after resource downloads or updates.  That said, we should avoid *directly*
-    // triggering an immediate reset, as an extra reset will occur once we leave the
-    // settings menu.  The delay also helps any chained downloads (keyboard > lexical model)
-    // to fully complete first.
-    shouldReloadKeyboard = true
```

Comment:
It turns out that this was a redundancy.  It's already set by [InstalledLanguagesViewController.swift](https://github.com/keymanapp/keyman/blob/8ea3f29e3a4071fc2b3a882b4e125d79f1880763/ios/engine/KMEI/KeymanEngine/Classes/Settings/InstalledLanguagesViewController.swift#L319)'s handler for the same notification.